src/utp_stream.cpp

```diff
@@ -2852,6 +2849,10 @@ bool utp_socket_impl::incoming_packet(boost::uint8_t const* buf, int size
 	if (ph->get_type() == ST_DATA)
 		m_sm->inc_stats_counter(counters::utp_payload_pkts_in);
 
+	// the number of packets that'll fit in the reorder buffer
+    // incoming MTU of 1100 and minimal size of 16 packets are chosen arbitrarly
+	boost::uint32_t max_packets_reorder = std::max(16, m_in_buf_size / 1100);
```

Comment:
this is a bit of a nit-pick, but I like to make anything that is not mutated ``const``

Reply:
changed to `const` and amend/force pushed the result. Is it even OK to --ament/ --force push things while targeting a pull request?

Reply:
yeah, that's great. It can make thinks more difficult to review if there are large changes, that's all